docker/package/packages.py

```diff
@@ -157,16 +157,25 @@
 """
 
```

Comment:
This commit (93b7817e7fb332451e8f1d3f7d8c5b4a71759b46) message has "instanted" in it, which seems a typo :slightly_smiling_face:

Reply:
Will update during the next rebase